scripts/update_firefox_os_feeds.py

```diff
@@ -0,0 +1,36 @@
+from django.conf import settings
+
+from feedparser import parse
+
+from bedrock.firefox.models import FirefoxOSFeedLink
+
+
+def create_or_update_fxos_feed_link(locale, link, title):
+    try:
+        feed_link = FirefoxOSFeedLink.objects.get(
```

Comment:
```python
feed_link, created = FirefoxOSFeedLink.objects.get_or_create(locale=locale, link=link, defaults={'title': title})
if not created and title != feed_link.title:
    feed_link.title = title
    feed_link.save()
```

Just a NIT. No need to change since you already typed it all out, just seemed like unnecessary repetition.